src/etl/allele_etl.py

```diff
@@ -151,267 +156,313 @@ class AlleleETL(ETL):
                     "uuid": str(uuid.uuid4()),
                     "dataProvider": data_provider,
-                    "symbolWithSpecies": alleleRecord.get('symbol') + " ("+ shortSpeciesAbbreviation + ")",
-                    "symbolTextWithSpecies": symbolText + " ("+ shortSpeciesAbbreviation + ")",
-                    "symbolText": symbolText,
+                    "symbolWithSpecies": allele_record.get('symbol') \
+                                         + " ("+ short_species_abbreviation + ")",
+                    "symbolTextWithSpecies": symbol_text + " ("+ short_species_abbreviation + ")",
+                    "symbolText": symbol_text,
                     "alleleDescription": description
                 }
 
                 alleles_no_constrcut_no_gene.append(allele_no_gene_no_construct_dataset)
 
             else:
-                logger.debug("ERROR: missing construct and gene")
+                self.logger.debug("ERROR: missing construct and gene")
 
 
-            if 'crossReferences' in alleleRecord:
+            if 'crossReferences' in allele_record:
 
-                for crossRef in alleleRecord['crossReferences']:
-                    crossRefId = crossRef.get('id')
-                    local_crossref_id = crossRefId.split(":")[1]
-                    prefix = crossRef.get('id').split(":")[0]
-                    pages = crossRef.get('pages')
+                for cross_ref in allele_record['crossReferences']:
+                    cross_ref_id = cross_ref.get('id')
+                    local_crossref_id = cross_ref_id.split(":")[1]
+                    prefix = cross_ref.get('id').split(":")[0]
+                    pages = cross_ref.get('pages')
 
                     # some pages collection have 0 elements
                     if pages is not None and len(pages) > 0:
                         for page in pages:
-                            if page == 'allele' or page == 'allele/references' or page == 'transgene' or page == 'construct' \
-                                    or page == 'transgene/references' or page == 'construct/references':
-                                modGlobalCrossRefId = ETLHelper.get_page_complete_url(local_crossref_id, self.xrefUrlMap, prefix, page)
-                                xref = ETLHelper.get_xref_dict(local_crossref_id, prefix, page, page, crossRefId, modGlobalCrossRefId, crossRefId+page)
-                                xref['dataId'] = globalId
-                                crossReferenceList.append(xref)
-
-            if 'synonyms' in alleleRecord:
-                for syn in alleleRecord.get('synonyms'):
+                            if page in ['allele', 'allele/references', 'transgene',
+                                        'construct', 'transgene/references',
+                                        'construct/references']:
+                                mod_global_cross_ref_id = ETLHelper.get_page_complete_url(\
+                                        local_crossref_id,
```

Comment:
can we include the first item in the list in the same line as the function call?  I find this more readable than the \.